install_config/prometheus_cluster_monitoring.adoc

```diff
@@ -0,0 +1,15 @@
+:toc: macro
+:toc-title:
+
```

Comment:
If you're making these modules, set the parent context here. The modules need context- friendly anchors to facilitate reuse, like `[id='concept-explanation-{context}']`